config.local.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Garradin;
+namespace Paheko;
```

Comment:
Should this be changed? I think they kept `namespace Garradin` in the code.
https://fossil.kd2.org/paheko/file?name=src/config.dist.php&ln=12

Reply:
Indeed, it souldn't be changed